prboom2/src/SDL/i_network.c

```diff
@@ -88,7 +91,11 @@ void I_ShutdownNetwork(void)
 void I_InitNetwork(void)
 {
   SDLNet_Init();
+#ifndef PRBOOM_SERVER
```

Comment:
Why does that need to be excluded?

Reply:
`prboom-plus-game-server` does not get linked with `SDL/i_main.c` and thus has no access to `I_AtExit()` - and wouldn't use it anyway, because is doesn't call `I_SafeExit()`.